### DISCUSSION TITLE
Next newsletter edition: How to use GitHub at work

### DISCUSSION THREAD

**Discussion by ettaboyle** [community/community#110340] [Announcements]
### Want this to be you? :nerd_face::point_down:
![use-github-at-work-like-the-office](https://github.com/community/community/assets/75342807/fdca9ac3-8571-49a5-853c-77f371e7533d)

Learn how to use GitHub to revolutionize your work life. From turbocharging your team's collaboration to beefing up security measures, we've got some tips and tricks to make each workday productive and enjoyable. :sparkles:

## ➡️ [Don’t miss it! Sign up now](https://resources.github.com/newsletter/)

**Comment by novium2** [comment]
Retiring this post as it is over a year old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
